src/Sylius/Bundle/LocaleBundle/Form/Type/LocaleChoiceType.php

```diff
@@ -40,7 +41,7 @@ public function configureOptions(OptionsResolver $resolver): void
         parent::configureOptions($resolver);
 
         $resolver->setDefaults([
-            'choices' => $this->localeRepository->findAll(),
+            'choices' => fn (Options $options): array => $this->localeRepository->findAll(),
```

Comment:
```suggestion
            'choices' => fn (): array => $this->localeRepository->findAll(),
```
It appears that the $option argument is not being utilized. Could you clarify the reason for including it here?

Reply:
The main reason I opted to include the `$options` is to mimic similar setups used in other classes:

CurrencyChoiceType:
https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/CurrencyBundle/Form/Type/CurrencyChoiceType.php#L40

ChannelChoiceType:
https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/ChannelBundle/Form/Type/ChannelChoiceType.php#L40

But I would not mind at all to drop it and implement your suggestion if preferred.

Reply:
If it is not a problem, please make it without the `options` parameter.

Reply:
I think `options` parameter is necessary even if it is not used